src/components/ChatLog.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+
+const ChatLog = ({entries, updateLikes}) => {
+    return (
+        <div className="chat-log">
+            {entries.map((chatEntry) => (
+                <ChatEntry
+                    key={chatEntry.id}
+                    sender={chatEntry.sender}
+                    body={chatEntry.body}
+                    timeStamp={chatEntry.timeStamp}
+                    liked={chatEntry.liked}
+                    updateLikes={() => updateLikes(chatEntry.id)}
```

Comment:
👀 The function you're passing in here for the like callback captures the id as part of the execution context, forgoing the need for the function to receive a parameter for the id. That's why this works (toggle like on the proper message) even though the id isn't passed down to the ChatEntry (and even though ChatEntry looks like it tries to use the id). Be sure to be consistent in your function definition and use. Since you define a function here that takes no parameters, then inside ChatEntry it should be called with no parameters. Or if you do want to pass the id in ChatEntry, this should continue to pass down the function it received from App as
```js
                    updateLikes={updateLikes}
```